ExcelReader/Services/FileUploadService.cs

```diff
@@ -10,11 +10,9 @@ namespace ExcelReader.Services
     public interface IFileUploadService
     {
         Task<bool> UploadExcelFile(IFormFile file);
-        Task<List<ITRequest>> GetExcelFileData();
+        Task<List<ItRequest>> GetExcelFileData();
         DateTime ConvertToDate(string dateString);
         Task<List<ITRequestWithFile>> GetITRequestsWithFiles();
```

Comment:
I think this method is not used anywhere.

Reply:
removed